cluster/manifests/node-problem-detector/ds-node-problem-detector.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: node-problem-detector
+  namespace: kube-system
+  labels:
+    application: node-problem-detector
+    version: v0.6.3
+spec:
+  selector:
+    matchLabels:
+      application: node-problem-detector
+      version: v0.6.3
+  template:
+    metadata:
+      labels:
+        application: node-problem-detector
+        version: v0.6.3
+    spec:
+      containers:
+      - name: node-problem-detector
+        image: k8s.gcr.io/node-problem-detector:v0.6.3
+        command:
+        - "/bin/sh"
```

Comment:
This doesn't evaluate any variables, so going via `sh` isn't really needed.

Reply:
everyone seems to run it like this for some reason. I changed to be more sane.